content/en/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -54,7 +54,6 @@ kubectl get pods -l run=my-nginx -o yaml | grep podIP
 
 You should be able to ssh into any node in your cluster and curl both IPs. Note that the containers are *not* using port 80 on the node, nor are there any special NAT rules to route traffic to the pod. This means you can run multiple nginx pods on the same node all using the same containerPort and access them from any other pod or node in your cluster using IP. Like Docker, ports can still be published to the host node's interfaces, but the need for this is radically diminished because of the networking model.
 
```

Comment:
```suggestion

You can read more about the [Kubernetes Networking Model](/docs/concepts/cluster-administration/networking/#the-kubernetes-network-model) if you're curious
```

Since the paragraph above the suggested deleted link talks about exposing pods from the networking perspective and ends with "the networking model", why don't we change the link to the Kubernetes Networking Model section instead of deleting the link

Reply:
Thanks @reylejano for reviewing and suggesting the change.